src/main/java/com/google/cloud/spanner/pgadapter/wireprotocol/QueryMessage.java

```diff
@@ -81,66 +75,40 @@ protected String getMessageName() {
   @Override
   protected String getPayloadString() {
     return new MessageFormat("Length: {0}, SQL: {1}")
-        .format(new Object[] {this.length, this.statement.getSql()});
+        .format(new Object[] {this.length, this.statement.getSqlWithoutComments()});
   }
 
   @Override
   protected String getIdentifier() {
     return String.valueOf(IDENTIFIER);
   }
 
-  public IntermediateStatement getStatement() {
+  public ParsedStatement getStatement() {
     return this.statement;
   }
 
-  /**
-   * Simple Query handler, which examined the state of the statement and processes accordingly (if
-   * error, handle error, otherwise sends the result and if contains result set, send row
-   * description)
-   *
-   * @throws Exception if handling the query fails
-   */
-  public void handleQuery() throws Exception {
```

Comment:
This is no longer needed. The simple query protocol now uses the same code as the extended query protocol. This is implemented by manually creating the parse-bind-describe-execute messages that would otherwise have been created for the extended protocol.